### PR TITLE
Update README to note the charm is deprecated

### PR DESCRIPTION
**Pull request by aieri**

### CODE REVIEW

**Comment by aieri** [comment]
Note that the markdown used is GH-specific: https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts

File: README.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # kubernetes-service-checks Charm
 
+> [!CAUTION]
+
```
Comment (samuelallan72) [inline README.md]: This doesn't render the icon (see [rich view](https://github.com/canonical/charm-kubernetes-service-checks/blob/a9ed4e27ba395d0e0da7feca72e083d177a64b3d/README.md)) - I'm guessing the blank line between this and the content needs to be removed?

File: README.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
...
+> This is a deprecated charm that only supports Kubernetes ≤ 1.28 and does not integrate with the Canonical Observability Stack. Please rely on native COS integration offered by the Kubernetes charms for monitoring Kubernetes deployments.
```
Comment (samuelallan72) [inline README.md]: It could be useful to add inline links to COS and the relevent docs for supported charms for monitoring k8s deployments.

**Review by samuelallan72 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (aieri) [reply README.md]: ugh yes you're right

**Review by samuelallan72 — APPROVED** [review]
Thanks, LGTM :)

**Review by rgildein — APPROVED** [review]

**Pull request merged**